src/es/lectortmo/src/eu/kanade/tachiyomi/extension/es/lectortmo/LectorTmoFactory.kt

```diff
@@ -1,17 +1,37 @@
-package eu.kanade.tachiyomi.extension.es.lectormanga
+package eu.kanade.tachiyomi.extension.es.lectortmo
 
-import eu.kanade.tachiyomi.multisrc.lectortmo.LectorTmo
 import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.NetworkHelper
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.SourceFactory
 import eu.kanade.tachiyomi.source.model.Page
 import eu.kanade.tachiyomi.source.model.SChapter
 import eu.kanade.tachiyomi.source.model.SManga
 import eu.kanade.tachiyomi.util.asJsoup
 import okhttp3.Response
 import org.jsoup.nodes.Document
 import org.jsoup.nodes.Element
+import uy.kohesive.injekt.injectLazy
 
-class LectorManga : LectorTmo("LectorManga", "https://lectormanga.com", "es") {
+class LectorTmoFactory : SourceFactory {
 
+    override fun createSources() = listOf(
+        LectorManga(),
+        TuMangaOnline(),
+    )
+}
+
+private val network: NetworkHelper by injectLazy()
+
+val rateLimitClient = network.cloudflareClient.newBuilder()
```

Comment:
the default client in LectorTmo have ratelimit host, is that intended? double rate limits?

Reply:
yep, otherwise site throws 429